docs/block-api.md

```diff
@@ -52,22 +52,34 @@ description: 'Block showing a Book card.'
 
 #### Category
 
-* **Type:** `String [ common | formatting | layout | widgets | embed ]`
+* **Type:** `String`
+
+Blocks are grouped into categories to help users browse and discover them. 
 
-Blocks are grouped into categories to help users browse and discover them. The core provided categories are `common`, `formatting`, `layout`, `widgets`, and `embed`.
+The core provided categories are:
+* common
+* formatting
+* layout
+* widgets
+* embed
 
 ```js
 // Assigning to the 'widgets' category
 category: 'widgets',
 ```
 
+Plugins & Themes can also register [custom block categories](../docs/block-api/extensibility.md).
```

Comment:
Why use the ampersand here over the word "and"?

Reply:
replaced with `and` will blame it on laziness :D